_targets.R

```diff
@@ -24,29 +25,24 @@ targets_list <- list(
     split_lake_centroids(centroids_sf_rds)
   ),
 
-  # Convert the lake centroids to some kind of polygon for querying GDP
-  tar_target(
-    query_polys_sf,
```

Comment:
Looks like this target is still used to build our test `query_map_png` below, FYI